Web/Scotty.hs

```diff
@@ -52,6 +52,7 @@ module Web.Scotty
       -- * Types
     , ScottyM, ActionM, RoutePattern, File, Content(..), Kilobytes, ErrorHandler, Handler(..)
     , ScottyState, defaultScottyState
+    , module Web.Scotty.Cookie
```

Comment:
I think explicit export list is more preferable because the type signatures show up in the Haddock.